packages/react-native-editor/src/index.js

```diff
@@ -10,6 +10,7 @@ import './globals';
 import { getTranslation } from '../i18n-cache';
 import initialHtml from './initial-html';
 import setupApiFetch from './api-fetch-setup';
+import { SETTINGS_DEFAULTS } from '../../block-editor/src/store/defaults.js';
```

Comment:
Hey @antonis, for imports from another module we usually would import `SETTINGS_DEFAULTS` like what's below. 

```suggestion
import { SETTINGS_DEFAULTS } from '@wordpress/block-editor';
```